src/output.cpp

```diff
@@ -2699,9 +2717,22 @@ std::pair<std::string, nc_color> rad_badge_color( const int rad )
     return std::pair<std::string, nc_color>( _( values[i].second.first ), values[i].second.second );
 }
 
-std::string formatted_hotkey( const std::string &hotkey, const nc_color text_color )
+std::string keybind_hint( const std::string &hotkey )
+{
+    // todo Add option to select between [x], x: and e[x]amine?
```

Comment:
FYI [this method of `input_context`](https://github.com/CleverRaven/Cataclysm-DDA/blob/dc9b4941013aafa6b8cae9dff13aa3695c5a75f2/src/input.cpp#L1115-L1120) have something similar to what you want here, but for registered input actions.